### DISCUSSION TITLE
Windows Install (.exe/.msi)

### DISCUSSION THREAD

**Discussion by Koyakami** [sct/overseerr#2806] [Q&A]
Hey all,

I already run a Windows 11 machine fine with Plex/Radarr/Sonarr/Ombi but this looks awesome, however I noticed all the installs were regarding Docker and I unfortunately don't run Docker.

I've considered it but my server runs flawless and easy to manage at current, any chance this Overseerr has a .exe/.msi I can run and it installs with a few clicks by any chance?

Thank you. (I did a search but people were referring to Docker on Windows for assistance)

**Comment by TheCatLady** [comment]
At this time, the only supported install method on Windows is Docker. You can subscribe to #1634 for updates.
